### ISSUE TITLE
The output on the console panel shows  messy code, why?

### ISSUE THREAD

**Issue by Imorton-zd** [fchollet/keras#578]
![qq 20150822095159](https://cloud.githubusercontent.com/assets/13865086/9422007/fbf706f0-48b3-11e5-98d5-a93fb182799e.png)

**Comment by EderSantana** [comment]
Can you show a code snippet of how you called the `fit` or `train_on_batch` method? If you are not doing that inside a for loop, while calling `print`, I'd say you may be missing a font.

**Comment by Imorton-zd** [comment]
Thanks. @EderSantana 
model.compile(loss='binary_crossentropy', optimizer='rmsprop', class_mode="binary")
model.fit(X_train, y_train, batch_size=batch_size, nb_epoch=nb_epoch,show_accuracy=True, validation_data=(X_test, y_test))
I have used more python compilers, all like this problem. How can I do? Thanks very much.

**Comment by fchollet** [comment]
It's basically an encoding problem. You might be able to configure your terminal to solve it (look for a "character encoding" option and select "utf-8").

I see you're on windows. I wonder if other windows users have had the same issue. Or maybe it's the combination of your locale + using windows that results in a non-standard default encoding.
